data/sound-strings/sk.json/localize.json

```diff
@@ -10,6 +10,9 @@
 "make_a_u_turn":"Otočte sa o stoosemdesiat stupňov.",
 "go_straight":"Choďte rovno.",
 "exit":"Výjazd.",
+"onto":"na",
```

Comment:
Please move all generated json files into a separate commit:
[strings] Regenerated TTS json files

Reply:
@biodranik This is done and should be ready for merge